app/helpers/application_helper.rb

```diff
@@ -64,4 +64,8 @@ def format_price(number)
   def kaminari_path(url)
     "#{root_url.chomp("\/")}#{url}"
   end
+
+  def user_text(text)
```

Comment:
Hi @iagirre! I think you can reuse `text_with_links` helper of `helpers/text_with_links_helper.rb` instead of create a new one 😉